products.css

```diff
@@ -34,6 +38,28 @@
     margin-top: -1.5rem;
     height: 100vh;
     background: radial-gradient(var(--white), var(--light-coral));
+    z-index: 5;
+}
+#toggle-sidebar {
+    visibility: hidden;
+}
+#toggle-sidebar:checked ~ .sidebar {
```

Comment:
using id for css styling is not the best practice

Reply:
Actually class was on working on it, that's why used id, please approve this PR